tokio-fs/src/remove_dir.rs

```diff
@@ -5,9 +5,7 @@ use std::path::Path;
 
 /// Removes an existing, empty directory.
 ///
-/// This is an async version of [`std::fs::remove_dir`][std]
-///
-/// [std]: https://doc.rust-lang.org/std/fs/fn.remove_dir.html
+/// This is an async version of [`std::fs::remove_dir`](std::fs::remove_dir)
```

Comment:
This is fine as is (no need to go back and fix all occurrences of this), but I think if you just use ``[`some::path`]`` it should link things properly given that the path resolves

Reply:
oh cool, I'll keep that in mind!